wordpress-mobile/WordPress-iOS
# Implement mentions in gutenberg by intercepting @ in the keyboard

Test triggering mentions using the keypress on @

GB PR: https://github.com/wordpress-mobile/gutenberg-mobile/pull/2424

To test:
 - Start the app
 - Create a new post on a blog/site on WP.com
 - Tap @ on the keyboard inside a paragraph block
 - Check that the mentions interface shows up
 - Check that selecting one user in the list inserts the mention in the block
 - Write a word and without adding a space type the @ on keyboard
 - Check that mentions suggestions doesn't show
 
 - Check that on self-hosted sites the mentions button in the toolbar is not shown, and the keypress of @ doesn't activate it.

PR submission checklist:
